app/views/flats/_nomad_carousel.html.erb

```diff
@@ -0,0 +1,33 @@
+<!-- Bootstrap HTML code for carousel on show page -->
+
+<div class="d-flex justify-content-center text-center">
+  <div id="carouselExampleControls" class="carousel slide" data-ride="carousel">
+    <div class="carousel-inner">
+      <div class="carousel-item active">
+        <% if @flat.photo.attachment %>
+          <%= cl_image_tag @flat.photo.key, alt: "First slide", class: "w-100" %>
+          <% else %>
+          <img class="d-block img-fluid" src="https://picsum.photos" alt="First slide">
+        <% end %>
+      </div>
+<!--         <div class="carousel-item">
+          <img class="d-block w-100" src="https://images.unsplash.com/photo-1489421382202-f7ec0cfd96f7?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1350&q=80" alt="Second slide">
+        </div>
+        <div class="carousel-item">
+          <img class="d-block w-100" src="https://images.unsplash.com/photo-1475260151973-4b51c9c9af2d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1352&q=80" alt="Third slide">
+        </div> -->
+    </div>
+    <!-- next & previous buttons on carousel -->
+      <a class="carousel-control-prev" href="#carouselExampleControls" role="button" data-slide="prev">
+        <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+        <span class="sr-only">Previous</span>
+      </a>
+      <a class="carousel-control-next" href="#carouselExampleControls" role="button" data-slide="next">
+        <span class="carousel-control-next-icon" aria-hidden="true"></span>
+        <span class="sr-only">Next</span>
+      </a>
+      <br>
+      <!-- flat address at bottom of carousel -->
+      <h4><%= @flat.address %></h4>
```

Comment:
I don't think we want to have the address on the carousel component. This should be kept in the old view

Reply:
Cool all changed